backend/api/views.py

```
@@ -367,6 +367,64 @@
+                video=video,
+                filename=filename,
+                recorder=request.user,
+                upload_status='pending'
+            )
+            
+            return Response({
+                "webcam_upload_url": webcam_upload_url,
+                "message": "Webcam recording upload URL generated successfully"
+            }, status=status.HTTP_201_CREATED)
+            
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)